base.css

```diff
@@ -0,0 +1,36 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+body {
+  margin: 25px;
+  border: 10px;
+  padding: 10px 20px;
+  border-style: solid;
+  border-radius: 5px;
+
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
```

Comment:
you are applying `font-family` here and aso in the universal selector `*` -and both are different.